cli/auth.go

```diff
@@ -206,6 +206,55 @@ func (c *viamClient) whoAmIAction(cCtx *cli.Context) error {
 	return nil
 }
 
+// OrganizationAPIKeyCreateAction corresponds to `organization api-key create`.
+func OrganizationAPIKeyCreateAction(cCtx *cli.Context) error {
+	c, err := newViamClient(cCtx)
+	if err != nil {
+		return err
+	}
+	if c.conf.Auth == nil {
+		return errors.New("not logged in: run the following command to login:\n\tviam login")
+	}
```

Comment:
[minor / q] can we somehow reuse `WhoAMIAction` here? 

Relatedly, might be worth it to update `whoAMIAction` to print the key ID if the auth type is api key.

Reply:
> Relatedly, might be worth it to update whoAMIAction to print the key ID if the auth type is api key.

Lets do that in the next pr where we actually add the ability to login with api keys.